src/appServiceNodes.ts

```diff
@@ -32,13 +32,17 @@ export class NodeBase {
 
 export class SubscriptionNode extends NodeBase {
     constructor(readonly subscription: SubscriptionModels.Subscription) {
-        super(`📰 ${subscription.displayName}`);
+        super(subscription.displayName);
     }
 
     getTreeItem(): TreeItem {
         return {
             label: this.label,
-            collapsibleState: TreeItemCollapsibleState.Collapsed
+            collapsibleState: TreeItemCollapsibleState.Collapsed,
+            iconPath: { 
+                light: path.join(__filename, '..', '..', '..', 'resources', 'light', 'AzureSubscription_16x.svg'),
+                dark: path.join(__filename, '..', '..', '..', 'resources', 'dark', 'AzureSubscription_16x.svg')
```

Comment:
The dark icon should be different. I can't seem to attach the file here, so I'll send an email

Reply:
Thanks, icon updated.